avocado/plugins/plugins.py

```diff
@@ -48,7 +48,9 @@ def run(self, args):
             (dispatcher.JobPrePostDispatcher(),
              'Plugins that run before/after the execution of jobs (job.prepost):'),
             (dispatcher.ResultDispatcher(),
-             'Plugins that generate job result in different formats (result):')
+             'Plugins that generate job result in different formats (result):'),
+            (dispatcher.ResultEventsDispatcher(args),
+             'Plugins that generate job result based on job/test events (result_events):')
```

Comment:
Also it'd be nice to split this long line. I know we don't mandate 80chars length, but it's better to keep it.